group_vars/all

```diff
@@ -66,6 +66,10 @@
   login_node_prefix: "login-"
   num_login_nodes: 0
 
+#OpenOnDemand
+  ood: "ood"
```

Comment:
Let's change this variable to ood_nodename.  It better describes its function and avoids confusion between a variable named ood and a value of ood.